homeassistant/components/webostv/media_player.py

```diff
@@ -313,16 +325,22 @@ def media_image_url(self) -> str | None:
     @property
     def supported_features(self) -> int:
         """Flag media player features that are supported."""
+        if self.state == STATE_OFF and self._supported_features is not None:
+            return self._supported_features
+
         supported = SUPPORT_WEBOSTV
 
-        if self._client.sound_output in ("external_arc", "external_speaker"):
+        if self._client.sound_output in (None, "external_arc", "external_speaker"):
```

Comment:
```suggestion
        if self._client.sound_output in ("external_arc", "external_speaker"):
```
I don't think we need `None` here anymore

Reply:
Why did we add `None` here?

Reply:
Now this is an interesting one... the reason I did this change was to get the most common denominator in use, and that would be not having `SUPPORT_VOLUME_SET`, hence the change I did!

I'm perfectly fine with reverting to the existing behavior if that is what you would prefer! 🙂

Reply:
What do you think @thecode?

Reply:
I think that once we restore the last state now we don't need the `None`.
The first check is for devices that don't support volume set and once it fails we fallback to a full set if the output is not `lineout` so basically we fallback to a full set in all cases except for 3 specific cases that probably don't support the full set.

Reply:
I've accepted the suggested change and reverted this back to the existing behavior!